src/engine/racer.rs

```diff
@@ -153,9 +142,6 @@ pub fn collapse_whitespace(text: &str) -> String {
   Regex::new(r"\s+").unwrap().replace_all(text, " ")
 }
 
-unsafe impl<'a> Sync for Racer<'a> {}
-unsafe impl<'a> Send for Racer<'a> {}
```

Comment:
I'm a bit surprised racerd still works without the `Send` impl. `FileCache` uses `Rc` internally and shouldn't actually be `Send`. We just have it here since we promise not to leak references outside the critical region. I guess travis seems ok with it...